util/zenfs.cc

```diff
@@ -420,7 +432,7 @@ int zenfs_tool_restore() {
 
 int zenfs_tool_dump() {
   Status s;
-  ZonedBlockDevice *zbd = zbd_open(true);
+  ZonedBlockDevice *zbd = zbd_open(true, true);
```

Comment:
Likewise, `zenfs dump` does not need exclusive access.
```suggestion
  ZonedBlockDevice *zbd = zbd_open(true, false);
```